stups-spring-oauth2-server/pom.xml

```diff
@@ -74,7 +74,7 @@
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-logging</artifactId>
+            <artifactId>spring-boot-starter-log4j</artifactId>
```

Comment:
it's not a step back, because we started with log4j in this project at least for the test, I configured it from the beginning to print http-client 'wire-log', but I realized the were gone during the build
and I want the 'wire-log' back  for the test, with this little change, it's much easier to explain people the oauth-dance if you can show the request-logs

so, don't think about this, it's just for testing